frontend/src/components/GoalsTable/GoalRow.js

```diff
@@ -0,0 +1,277 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+// import { Link, useHistory } from 'react-router-dom';
+import moment from 'moment';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faClock, faCheckCircle, faExclamationCircle, faPencilAlt, faMinusCircle, faTimesCircle, faFlag,
+} from '@fortawesome/free-solid-svg-icons';
+import ContextMenu from '../ContextMenu';
+import Tooltip from '../Tooltip';
+import { DATE_DISPLAY_FORMAT } from '../../Constants';
+import { reasonsToMonitor } from '../../pages/ActivityReport/constants';
+import { updateRecipientGoalStatus } from '../../fetchers/recipient';
+import './GoalRow.css';
+
+function GoalRow({
+  goal,
+  openMenuUp,
+  updateGoal,
+}) {
+  const {
+    id,
+    goalStatus,
+    createdOn,
+    goalText,
+    goalTopics,
+    objectiveCount,
+    goalNumber,
+    reasons,
+  } = goal;
+
+  const [trClassname, setTrClassname] = useState('tta-smarthub--goal-row');
+
+  /* TODO: Setup Route for Edit Goal.
+  /*
+  const history = useHistory();
+  // eslint-disable-next-line max-len
+  const viewOrEditLink =
+    calculatedStatus === 'approved'
+    ? `/activity-reports/view/${id}`
+    : `/activity-reports/${id}`;
+  */
+
+  const contextMenuLabel = `Actions for goal ${id}`;
+
+  /**
+   * we manage the class of the row as a sort of "focus-within" workaround
+   * this is entirely to show/hide the export reports button to keyboard users but
+   * not blast screen-reader only users with a bunch of redundant buttons
+   */
+  const onFocus = () => setTrClassname('tta-smarthub--goal-row focused');
```

Comment:
are we using the focused class for anything? I'm pretty sure its in place for that export reports button as well, so you can probably remove the class name from state and remove these focus handlers

Reply:
Removed focus code.